app/helpers/measures_helper.rb

```diff
@@ -2,4 +2,8 @@ module MeasuresHelper
   def order_number_info(record)
     record.definition.try(:description) || "Order number #{record.number}"
   end
+
+  def filter_duty_expression(record)
+    (record.eql? 'NIHIL') ? '' : record
```

Comment:
as a personal preference, I would use `==`

Reply:
Ok, don't merge it yet and let me change it to ==